tvac/cli.py

```diff
@@ -0,0 +1,10 @@
+import argparse
+from .client import VariantAnnotationClient
+
+def cli_annotate_variants():
```

Comment:
In a perfect world, I would probably have a minor preference for the cli tool formatting to use `tvac {command} {args}` rather than have a command specifically for annotate-variants.